components/hook-http/node/index.mjs

```diff
@@ -1,14 +1,16 @@
 const {
   Buffer: { from: toBuffer, concat: concatBuffer },
-  Object: { fromEntries },
+  Object: { fromEntries, entries: toEntries },
   TextDecoder,
   Reflect: { apply },
   JSON: { parse: parseJSON },
+  Math: { round },
   undefined,
 } = globalThis;
```

Comment:
For my own edification: I understand the renaming that's happening here, but why are you destructuring `TextDecoder` and `undefined` from `globalThis`? Can't you just reference in the code without qualification?

Reply:
This to protect against the user changing the global object. It is to separate as much as possible the observed application layer from the agent layer.